packages/l/libvorbis/xmake.lua

```diff
@@ -10,13 +10,66 @@ package("libvorbis")
 
     add_versions("1.3.7", "0e982409a9c3fc82ee06e08205b1355e5c6aa4c36bca58146ef399621b0ce5ab")
 
+    add_configs("vorbisenc",  {description = "Includes vorbisenc", default = true, type = "boolean"})
+    add_configs("vorbisfile", {description = "Includes vorbisfile", default = true, type = "boolean"})
+
     add_deps("cmake", "libogg")
 
+    on_fetch(function (package, opt)
+        if opt.system then
+            local vorbis = find_package("vorbis")
+            if not vorbis then
+                return
+            end
+            local result = table.copy(vorbis)
+
+            if package:config("vorbisenc") then
+                local vorbisenc = find_package("vorbisenc")
+                if not vorbisenc then
+                    return
+                end
+
+                result.includedirs = table.join(vorbisenc.includedirs, result.includedirs or {})
```

Comment:
and sysincludedirs

Reply:
Should I replace includedirs by sysincludedirs or add a line to join sysincludedirs?

Reply:
join(includedirs or sysincludedirs)